cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetInstanceService.java

```diff
@@ -472,12 +477,8 @@ private void disableExplore(DatasetId datasetInstance) {
     try {
       exploreFacade.disableExploreDataset(datasetInstance.toId());
     } catch (Exception e) {
-      String msg = String.format("Cannot disable exploration of dataset instance %s: %s",
-                                 datasetInstance, e.getMessage());
-      LOG.error(msg, e);
+      LOG.error("Cannot disable explore for dataset instance {}: {}", datasetInstance, e.getMessage(), e);
```

Comment:
`e.getMessage` seems redundant, since we're passing `e` as the last parameter.
Won't the message get logged twice now?